src/Admin/SupportScreen.php

```diff
@@ -106,6 +106,7 @@ public static function is_needed() {
 		return (
 			is_admin()
 			&&
+			// Note: The view_site_health_checks capability was introduced in WordPress 5.2, so this will return false in older versions.
```

Comment:
Even the `view_site_health_checks` capability introduces in WordPress 5.2 but that is available in the "Health Check & Troubleshooting" plugin. So it will enable the "Support" screen.

However, the Support page is working fine in WordPress 4.9

Reply:
Added fix in [c8a2d41](https://github.com/ampproject/amp-wp/pull/6773/commits/c8a2d418f05ef0cd11d264ea6392cb1015f4f7c9)

Reply:
When Health Check & Troubleshooting is enabled, the Support page works fine in 4.9?

Reply:
The Support page is working fine in 4.9 However as of now the value of the loopback request is not coming properly and to address that we need to use a function from the "Health Check & Troubleshooting" plugin.